packages/client/components/ArchiveTaskRoot.tsx

```diff
@@ -4,18 +4,34 @@ import {QueryRenderer} from 'react-relay'
 import useAtmosphere from '~/hooks/useAtmosphere'
 import useDocumentTitle from '~/hooks/useDocumentTitle'
 import TeamArchive from '~/modules/teamDashboard/components/TeamArchive/TeamArchive'
-import {LoaderSize} from '~/types/constEnums'
-import renderQuery from '~/utils/relay/renderQuery'
+import UserTasksHeader from '~/modules/userDashboard/components/UserTasksHeader/UserTasksHeader'
+import ErrorComponent from './ErrorComponent/ErrorComponent'
 
 
 const query = graphql`
   query ArchiveTaskRootQuery($first: Int!, $after: DateTime, $userIds: [ID!], $teamIds: [ID!]) {
     viewer {
+      ...UserTasksHeader_viewer
       ...TeamArchive_viewer
     }
   }
 `
 
+const renderQuery = ({error, props}) => {
+  if (error) {
+    return <ErrorComponent error={error} eventId={''} />
+  }
+  if (!props) {
+    return <UserTasksHeader viewer={null} />
+  }
+  return (
+    <>
+      <UserTasksHeader viewer={props.viewer} />
+      <TeamArchive viewer={props?.viewer ?? null} returnToTeamId={props?.returnToTeamId} team={props?.team} />
```

Comment:
under what condition is `team` not undefined here?
hint: move this inside the functional component, you're not passing team in where you should be